_includes/markdown/PHP.md

```diff
@@ -88,7 +88,46 @@ Here are a few key points:
 
 * A [taxonomy](https://wordpress.org/support/article/taxonomies/) is a tool that lets us group or classify posts.
 
-    [Post meta](https://wordpress.org/support/article/custom-fields/) lets us store unique information about specific posts. As such the way post meta is stored does not facilitate efficient post lookups. Generally, looking up posts by post meta should be avoided (sometimes it can't). If you have to use one, make sure that it's not the main query and that it's cached.
+*    [Post meta](https://wordpress.org/support/article/custom-fields/) lets us store unique information about specific posts. As such the way post meta is stored does not facilitate efficient post lookups. Generally, looking up posts by post meta should be avoided (sometimes it can't). If you have to use one, make sure that it's not the main query and that it's cached.
+
+* Always set default values for post_meta keys, especially if they are used to filter other queries. When writing queries filtering by post meta, never use the compare type ```NOT EXISTS```.
```

Comment:
@moraleida Could you put an example of when we'd set default values for post_meta keys? Is that when using update_post_meta or add_post_meta? I feel like we need a bit more information on why default values (instead of empty strings, I assume) are a best practice.

Reply:
I'd recommend we lead with 'Never use compare type NOT EXISTS' as the leading part of this (or on its own line) since it tees up the majority of the rest of the section